cmd/livepeer/livepeer.go

```diff
@@ -187,6 +189,17 @@ func main() {
 		n.NodeType = core.BroadcasterNode
 	}
 
+	if n.NodeType == core.BroadcasterNode {
+		if *orchAddr == "" {
+			glog.Info("No orchestrator specified; transcoding will not happen")
```

Comment:
Can we exit here for now?  In the next iteration when we implement O discovery via onchain info, we can take it out.

Reply:
We can do that, but it'd mean stopping support for view-only source streams.

Although I suppose we might also want to figure out how to support view-only mode once we use onchain info.

Reply:
Ahh I see.  Ok I'm on board for keeping things this way then.  view-only feels useful.